benchmarks/logic.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from sympy.core.compatibility import range
+from sympy.logic.utilities import load_file
+from sympy.logic import satisfiable
+
+input_path = os.path.dirname(__file__)
+
+
+class LogicSuite:
+    """
+    Benchmark suite for sympy.logic
+    """
+    def setup(self):
+        self.INPUT = [5 * i for i in range(2, 16)]
+        self.ALGORITHMS = ['dpll', 'dpll2']
+
```

Comment:
Maybe you want to see differences between different algorithms? If so you can introduce `time_dpll(self)` and `time_dpll2(self)`